day-10-of-30/Power of 2/C++/yash1402_.cpp

```diff
@@ -0,0 +1,18 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+  int n;
+  cin >> n;
+  
+  if (n == 0) {
+    cout << "false" << endl;
+    return 0;
+  }
+  
+  int sq = sqrt(n);
+  if (sq * sq == n) cout << "true" << endl;
```

Comment:
- We want to return `true` if `n = 2^k, where k = 0,1,2,3,4,......... ` else return `false`.
- And you were written the logic of if the number is **perfect square** or not?
- I think there are some edge test cases that are not given so your code is accepted.
- Kindly checkout the logic.

Reply:
Fixing this in next pull request

Reply:
OK